### PR TITLE
[15.0][MIG] account_due_list: Migration to 15.0

### PR DESCRIPTION
**Pull request by LudLaf**
Standard migration to version 15

@Tecnativa 

TT35672

### CODE REVIEW

File: account_due_list/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -1,3 +1,4 @@
-To use this module, you need to :
+To use this module, you need to go to:
```
Comment (pedrobaeza) [inline account_due_list/readme/CONFIGURE.rst]: Please put several lines with the steps, instead all of them together. We use `#. ` for auto-numbering them.

Comment (LudLaf) [reply account_due_list/readme/CONFIGURE.rst]: @pedrobaeza & @victoralmau please review this PR.

@Tecnativa

TT35672

**Review by victoralmau — APPROVED** [review]
Code review and UI ok.
Maybe you can squash administrative commits (to reduce the number of commits) and purge translations.

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Comment by LudLaf** [comment]
> 



> Code review and UI ok. Maybe you can squash administrative commits (to reduce the number of commits) and purge translations.

​@victoralmau​ ​thanks for your suggestion. I believe it's too late for me to make those changes, but I​ wi​ll take it into account from now on.​​

**Pull request merged**